jetty-servlet-api/pom.xml

```diff
@@ -108,6 +82,54 @@
         </executions>
       </plugin>
 
+      <!-- Compile all the sources -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.0</version>
+        <configuration>
+          <source>11</source>
```

Comment:
do we really need to enforce java11? the general build for those projects use java8 as well? 
otherwise we need to change the Jenkinsfile

Reply:
`jetty-servlet-api` will only be used in Jetty 10, so yes JDK 11.